### ISSUE TITLE
[FEAT] Right-click context menus for pinned apps

### ISSUE THREAD

**Issue by ThePixelDude111** [eythaann/Seelen-UI#130]
It would be a nice addition to version 2.0, and maybe add it to running apps too.

**Comment by eythaann** [comment]
apps on dock already have a context menu

**Comment by ThePixelDude111** [comment]
> apps on dock already have a context menu

Sorry, I meant the app-specific shortcuts, like web browsers of the file explorer itself,
like this:
![Screenshot 2024-10-11 162210](https://github.com/user-attachments/assets/205d35b1-8531-40bf-8964-814effb07a2c)

**Comment by ThePixelDude111** [comment]
Yeah and this is not windows 11, this is just windows 10 with rectify11 (22h2)

**Comment by eythaann** [comment]
ohhh I see, change the title of the issue and add a more detailed description of the expected behavior I will try to make this.

**Comment by IVIJL** [comment]
yeah this I miss a lot, thyat I have pinned shortcuts to folder on right click also right click on start button not showing all the settings, which if it is not possible would be nice to have this menu on the top left under my name on left click somewhere these are most used right click actions I use on windows and blocking me from using this :(
